src/OneSignal.ts

```diff
@@ -373,6 +373,16 @@ export default class OneSignal {
     await OneSignal.context.promptsManager.internalShowSlidedownPrompt(options);
   }
 
+  public static async showCategorySlidedown(options?: AutoPromptOptions): Promise<void> {
+    await awaitOneSignalInitAndSupported();
+    const isPushEnabled = await OneSignal.privateIsPushNotificationsEnabled();
+    await OneSignal.context.promptsManager.internalShowCategorySlidedown({
+      ...options,
+      force: true,
```

Comment:
Please add an in-code comment about `force: true,`

Reply:
You're right this is no longer needed

Reply:
Done.